packages/viewer/src/components/viewer-markup-arrow/viewer-markup-arrow.spec.tsx

```diff
@@ -0,0 +1,53 @@
+jest.mock('../viewer/utils');
+jest.mock('../viewer-markup/dom');
+
+import '../../testing/domMocks';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
nit: Can this be removed?

Reply:
Not without producing a lint warning. Unfortunately, it's a side-effect of how the Stencil compiler works.